desktop/app/kbfs-helper.js

```diff
@@ -1,10 +1,32 @@
 // @flow
 
 import {ipcMain, shell} from 'electron'
+import fs from 'fs'
+import {pathToURL} from './paths'
 
 export default function () {
   ipcMain.on('openInKBFS', (e, path) => {
-    // We do the `shell.openItem` here to not block the ui thread
-    shell.openItem(path)
+    fs.stat(path, (err, stats) => {
```

Comment:
This hangs my electron instance when checking a path that KBFS hasn't accessed recently. I'm a tad surprised by that because it appears to be a non-blocking call to `fs.stat`, but I've been able to reproduce it several times.